src/main/openapi/discovery.yaml

```diff
@@ -235,19 +250,9 @@ components:
         contact:
           $ref: "#/components/schemas/ValueLabel"
           title: Contact
-        hasVersions:
```

Comment:
important: please review if we are using this property in the frontend. If yes, please revert this change and part this property as deprecated instead.

Reply:
it will be deleted in the front end too. This is a relation which goes into relations.